### DISCUSSION TITLE
How to hide bootstrap license and select folder on minor upgrade

### DISCUSSION THREAD

**Discussion by mjanulaitis** [wixtoolset/issues#7146] [Questions]
In Wix3 when using the internal MSI UI the license agreement and select folder options are not displayed on a minor upgrade.  I have moved the UI to burn however now during a minor upgrade the license and select folder options are still displayed.  How can I hide these options and make sure the upgrade updates the files in the user's overridden install folder?

**Comment by rseanhall** [comment]
If you moved the UI to burn then the answer depends on which Bootstrapper Application you're using.
